### ISSUE TITLE
Android - Search - VoiceOver Does Not Announce Search Results after Searching

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#165]
While using TalkBack, the user is not notified that any results were found after tapping the Search button.

When I searched for the word "test," it found ten screens that had it on.

**Comment by jfc3-dol** [comment]
I was not able to test this because I could not enter search terms suing TalkBack, which is noted in issue #158 - Android - Not Able to Search when Using TalkBack.

**Comment by jfc3-dol** [comment]
@pbhatt17

Using Talkback on the ILAB Android Comply chain application, I'm not able to type anything in the search field, so I'm not able to test if this issue is fixed or not.

**Comment by jfc3-dol** [comment]
@pbhatt17

While using Talkback on the ILAB Android Comply Chain application, I'm not able to type anything in the search field, so I'm not able to test if this issue is fixed or not.

**Comment by jfc3-dol** [comment]
@pbhatt17 

On either of our test devices, I am not able to type a search term using TalkBack.

**Comment by jfc3-dol** [comment]
@pbhatt17

Because I can not search using TalkBack on our test devices I am not able to type a search term.
